### ISSUE TITLE
Latexml does not like `\input{"Path with spaces"}`

### ISSUE THREAD

**Issue by FelixBenning** [brucemiller/LaTeXML#1847]
Because I am experimenting with templates, I ended up pasting an absolute path into an input macro before passing it to latexml. Narrowing down where the errors come from I realized that latexml does not like spaces in paths while latex itself was able to render the file fine.

i.e.

```latex
\input{"/Users/felix/Library/CloudStorage/OneDrive-SharedLibraries-uni-mannheim.de/Team Stochastik - 2021/Tex-Files/exercises/tightness.tex"}
```
is accepted by latexmk but not latexml. Given that quotes are the [suggested way to deal with spaces](https://tex.stackexchange.com/questions/191930/spaces-in-files-names-in-input-or-includegraphics) it seems like this should probably work

**Comment by dginev** [comment]
This is curious, since we have previously added support for both quoted string literals and spaces.

Could I ask you @FelixBenning  to also double-check if the file in question (and the directory pieces to it) are in the same case as the TeX input string? E.g. that it is really `Tex-` and not `TeX-` etc.

We technically have also added some support for case-insensitive file systems, but I'd imagine that is more likely to still be fragile. Since we don't have your file system to test with, the best I can do is ask. Thanks!

**Comment by FelixBenning** [comment]
```
(env) felix@stochastik7 Tex-Files % pwd
/Users/felix/Library/CloudStorage/OneDrive-SharedLibraries-uni-mannheim.de/Team Stochastik - 2021/Tex-Files
```

okay so it appears

```latex
\documentclass[11pt]{article}
\usepackage{exerciseHead}

%%%%%%%%% INCLUDE SOLUTIONS? %%%%%%%%%% 
\solutiontrue
%%%%%%%%%%%%
\input{Macros.tex}


\begin{document}
	\input{"/Users/felix/Library/CloudStorage/OneDrive-SharedLibraries-uni-mannheim.de/Team Stochastik - 2021/Tex-Files/exercises/ball_bag.tex"}
\end{document}
```

compiles in latex but not with `latexmlc test.tex`. Removing the quotes results in successful compilation.

**Comment by brucemiller** [comment]
I *think* this got fixed (maybe 4a74b0b ?).  At least it seems to work now.  If I'm wrong, please reopen. Thanks for the report!
